- enable conda-rdkit builds on all platforms

ptosco opened the pull request:
Enables rdkit, rdkit-postgresql95 and rdkit-postgresql conda builds on all platforms.
I have tested it on the latest Anaconda release and it works on all platforms.
Please note that this PR requires https://github.com/rdkit/rdkit/pull/1455, https://github.com/rdkit/rdkit/pull/1456 and https://github.com/rdkit/rdkit/pull/1457.
Also note git points to https://github.com/ptosco/rdkit, but it should be reverted to https://github.com/ptosco/rdkit once the pull requests are merged in. The same applies to RDkit revisions.

greglandrum commented:
Hmmm, I must be doing something stupid (always likely under Windows) because I can't get anything sensible to happen when I do `conda build rdkit-postgresql95`. It looks like there's no bld.bat there. Did you forget to upload those?

ptosco commented:
Oops, sorry Greg, you are not doing anything stupid. It looks like when I split my local development branch into multiple branches to keep PRs separate I left out those files. I have just added them.

greglandrum commented:
Thanks. I'm now able to build rdkit-postgresql95 on my windows laptop.
I can't do rdkit-postgresql since the postgresql builds themselves don't work (missing `gettext` dependency), but I don't see that as critical.

ptosco commented:
Hi Greg, I have just submitted a new PR with the Windows PostgreSQL
dependencies. You should now be able to also build rdkit-postgresql; please
shout if this is not the case.

On Fri, Jun 30, 2017 at 4:21 AM, Greg Landrum <notifications@github.com>
wrote:

> Merged #51 <https://github.com/rdkit/conda-rdkit/pull/51>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rdkit/conda-rdkit/pull/51#event-1145321733>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AFAF4aChp2l4cyeSJbnuKJIAxt3ePf-mks5sJGmxgaJpZM4N6T62>
> .
>
